### DISCUSSION TITLE
Clarification about ESLZ design principle

### DISCUSSION THREAD

**Discussion by TarekNabeel** [Azure/Enterprise-Scale#954] [General] [answered]
I need more clarification about the design principle "Application-centric and archetype-neutral" , what is actually meant by "application-centric" and "archetype-neutral" as acronyms and also i need an example of this principle if possible.

**Comment by jtracey93** [comment, marked as answer]
Hey @TarekNabeel,

Have you seen https://docs.microsoft.com/azure/cloud-adoption-framework/ready/enterprise-scale/design-principles#application-centric-and-archetype-neutral ?

**Reply by TarekNabeel** [reply]
@jtracey93 , thanks yes i wen through it but still not clear

**Reply by jtracey93** [reply]
Okay cool.

Effectively it means that ALZ/ESLZ doesn't lean towards supporting an IaaS only or PaaS only model, it supports both as it is agnostic to the type of workload.

It also is about focussing on the application and supporting it and its teams with what they need to move quickly by enabling their autonomy by configuring the ALZ/ESLZ platform in such a way - e.g. policy-driven governance.

Does this help @TarekNabeel?

**Comment by TarekNabeel** [comment]
@Jack , apologies for late response and many thanks for the clear
clarification.
